menu/cbs/menu_cbs_sublabel.c

```diff
@@ -1803,6 +1806,17 @@ static int action_bind_sublabel_generic(
    return 0;
 }
 
+#ifdef HAVE_MIST
+static int action_bind_sublabel_core_manager_steam_entry(
```

Comment:
This function is unnecessary - if an entry has no sublabel, then just omit the `BIND_ACTION_SUBLABEL()`